Tests/Integration/Components/CategoryResolver/AutoCategoryResolverTest.php

```diff
@@ -79,7 +79,7 @@ public function testResolve()
         ]);
 
         if (!$nikeBootsCategory) {
-            $this->categoryResolver->createLocalCategory('Boots', '/deutsch/nike/boots', $nikeCategoryId, 1234);
+            $this->categoryResolver->createLocalCategory('Boots', '/deutsch/nike/boots', $nikeCategoryId, 1234, 'test');
```

Comment:
same here and all other places where we have 'test' for stream name. My opinion is that we need to use similar data with production, as much as possible.